hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -1,38 +1,34 @@
 import asyncio
-import json
 import logging
 import time
 from collections import namedtuple
 from decimal import Decimal
 from enum import Enum
-from typing import (
-    Any,
-    AsyncIterable,
-    Dict,
-    List,
-    Optional,
-)
-
-import aiohttp
+from typing import Any, AsyncIterable, Dict, List, Optional
 
 from hummingbot.connector.client_order_tracker import ClientOrderTracker
-from hummingbot.connector.exchange.ascend_ex import ascend_ex_constants as CONSTANTS, ascend_ex_utils
+from hummingbot.connector.exchange.ascend_ex import ascend_ex_constants as CONSTANTS
+from hummingbot.connector.exchange.ascend_ex import ascend_ex_utils
+from hummingbot.connector.exchange.ascend_ex.ascend_ex_api_order_book_data_source import \
+    AscendExAPIOrderBookDataSource as OBDS
```

Comment:
It is not a good idea to use not easy to read aliases. It makes the code more difficult to understand.